### ISSUE TITLE
What are the specific implementation values to reproduce your result?

### ISSUE THREAD

**Issue by hleu** [lengstrom/fast-style-transfer#161]
I just want to first reproduce your result on the chicago.jpg and rain_princess.jpg but the default settings does not give me a result even close at all. Can you please let me know what are the parameters that you used with style.py to get the training result like what you have? Thank you!
(Specifically number of epochs, learning rate, etc.)

**Comment by spot92** [comment]
Did you find out?
